doc/dev_start_guide.txt

```diff
@@ -267,41 +252,37 @@ You can use ``easy_install`` instead of ``pip``, and ``pep8`` instead of
 ``flake8`` if you prefer. The important thing is that the ``flake8`` or
 ``pep8`` executable ends up in your ``$PATH``.
 
-To install Syntastic, according to its documentation, the easiest way is
-to install `pathogen.vim`_ first.
-
-Here's a relevant extract of pathogen.vim's installation instructions:
-
-    Install to ``~/.vim/autoload/pathogen.vim``. Or copy and paste::
-
-        mkdir -p ~/.vim/autoload ~/.vim/bundle; \
-        curl -so ~/.vim/autoload/pathogen.vim \
-            https://raw.github.com/tpope/vim-pathogen/HEAD/autoload/pathogen.vim
+Install vundle with: ``git clone https://github.com/VundleVim/Vundle.vim.git
```

Comment:
I edited the previous comment to fix a typo. Refer to the latest version please.